README.md

```diff
@@ -0,0 +1,31 @@
+# ARRAY SCALER
+
+## Introduction
+A simple python tool for scaling and resvaling arrays:
```

Comment:
I am not sure I know what `resvaling` is or is it a typo?

Reply:
Good catch!!!. This is actually a typo and I would fix it